src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.exception.LottoException;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        validateDuplicate(numbers);
+        validNumber(numbers);
+        this.numbers = orderByASC(numbers);
+    }
+
+    public List<Integer> getNumbers() {
+        return numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != 6) {
+            throw new IllegalArgumentException(LottoException.INVALID_LOTTO_SIZE);
+        }
+    }
+
+    private void validateDuplicate(List<Integer> numbers) {
+        if (numbers.size() != numbers.stream().distinct().count()) {
+            throw new IllegalArgumentException(LottoException.DUPLICATE_LOTTO_NUMBER);
+        }
+    }
+
+    private void validNumber(List<Integer> numbers) {
+        for (Integer number : numbers) {
+            if (number < 0 || number > 45) {
+                throw new IllegalArgumentException(LottoException.INVALID_LOTTO_NUMBER_RANGE);
+            }
+        }
+    }
+
+    private List<Integer> orderByASC(List<Integer> numbers) {
+        return numbers.stream().sorted().toList();
```

Comment:
stream의 장점 중 하나는 각 단계를 메서드로 구분하여 사람의 생각 흐름과 코드의 흐름을 비슷하게 작성할 수 있다는 것이에요.
각 메서드들 사이에 개행을 포함한다면 훨씬 가독성이 좋아집니다.
```suggestion
        return numbers.stream()
                .sorted()
                .toList();
```